src/clj/chronograph/handlers/google_auth.clj

```diff
@@ -0,0 +1,72 @@
+(ns chronograph.handlers.google-auth
+  (:require [ring.util.codec :as codec]
+            [ring.util.response :as response]
+            [chronograph.config :as config]
+            [chronograph.auth :as auth]
+            [chronograph.db.users :as users-db]
+            [org.httpkit.client :as http]
+            [mount.core :refer [defstate]]
+            [cheshire.core :as json])
+  (:import (com.google.api.client.googleapis.auth.oauth2 GoogleIdTokenVerifier$Builder GoogleIdTokenVerifier)
+           (com.google.api.client.googleapis.javanet GoogleNetHttpTransport)
+           (com.google.api.client.json.jackson2 JacksonFactory)))
+
+(defstate google-id-token-verifier
+  :start (-> (GoogleIdTokenVerifier$Builder. (GoogleNetHttpTransport/newTrustedTransport)
+                                             (JacksonFactory.))
+             (.setAudience [(get-in config/config [:oauth :google :client-id])])
+             (.build))
+  :stop nil)
+
+(defn token->credentials
```

Comment:
The name doesn't seem to indicate that it can fail. Should we call this `verify`?